std_msgs/Header header
int32 imgFrameId
int32 sequencerSetIndex
float32 exposureTime
